bokeh/models/widgets/tables.py

```diff
@@ -1,740 +1,764 @@
+    =========== =============== =============
+    1000.234    '$0,0.00'       $1,000.23
+    1000.2      '0,0[.]00 $'    1,000.20 $
+    1001        '$ 0,0[.]00'    $ 1,001
+    -1000.234   '($0,0)'        ($1,000)
+    -1000.234   '$0.00'         -$1000.23
+    1230974     '($ 0.00 a)'    $ 1.23 m
+    =========== =============== =============
+
+    **BYTES**:
+
+    =============== =========== ============
+    Number          Format      String
+    =============== =========== ============
+    100             '0b'        100B
+    2048            '0 b'       2 KB
+    7884486213      '0.0b'      7.3GB
+    3467479682787   '0.000 b'   3.154 TB
+    =============== =========== ============
+
+    **PERCENTAGES**:
+
+    ============= ============= ===========
+    Number        Format        String
+    ============= ============= ===========
+    1             '0%'          100%
+    0.974878234   '0.000%'      97.488%
+    -0.43         '0 %'         -43 %
+    0.43          '(0.000 %)'   43.000 %
+    ============= ============= ===========
+
+    **TIME**:
+
+    ============ ============== ============
+    Number       Format         String
+    ============ ============== ============
+    25           '00:00:00'     0:00:25
+    238          '00:00:00'     0:03:58
+    63846        '00:00:00'     17:44:06
+    ============ ============== ============
+
+    For the complete specification, see http://numbrojs.com/format.html
+    """)
+
+    language = Enum(NumeralLanguage, default="en", help="""
+    The language to use for formatting language-specific features (e.g. thousands separator).
+    """)
+
+    rounding = Enum(RoundingFunction, help="""
+    Rounding functions (round, floor, ceil) and their synonyms (nearest, rounddown, roundup).
+    """)
+
+class BooleanFormatter(CellFormatter):
+    ''' Boolean (check mark) cell formatter.
+
+    '''
+
+    icon = Enum('check', 'check-circle', 'check-circle-o', 'check-square', 'check-square-o', help="""
+    The icon visualizing the check mark.
+    """)
+
+class CompositeFormatter(CellFormatter):
+    '''CompositeFormatter applies multiple formatters to a single TableColumn
+
+    Example:
+    .. code-block:: python
+
+        example_formatter = CompositeFormatter(formatters=[StringFormatter(font_style="bold"), HTMLTemplateFormatter(
+        template='<code><%= value %></code>')])
+
+        TableColumn(field='Part ID', title='Part ID', formatter=example_formatter, ...)
+    '''
+
+    formatters = Seq(Instance(CellFormatter, help="""
+        The CompositeFormatter class may be used with any combination of
+        the installed formatters in the Bokeh library to apply multiple
+        formatting capabilities (BooleanFormatter, CellFormatter, DateFormatter,
+        HTMLTemplateFormatter, NumberFormatter, ScientificFormatter, StringFormatter,
+        to a single TableColumn. Formatters are implemented in the order they are 
+        listed, i.e: from "(" to ")".

```

Comment:
There may not be `(` e.g. if the sequence is a list, or an array of objects, so that might be confusing. Suggest:
> Formatter are applied in the order they appear in the sequence.